pkg/fuzzy/google_translate.go

```diff
@@ -271,7 +271,7 @@ func patternToString(pattern []ast.Pattern) string {
 		switch v := pattern[i].(type) {
 		case ast.TextPattern:
 			text += string(v)
-		case ast.PlaceholderPattern:
+		case ast.LiteralExpression, ast.AnnotationExpression, ast.VariableExpression, ast.Markup: // or default
```

Comment:
that default comment was for you :), shouldnt be here
either
1. case ast.LiteralExpression, ast.Annotation...
2. default